app/src/pages/project/SessionsTable.tsx

```diff
@@ -0,0 +1,305 @@
+  getSortedRowModel,
+  SortingState,
+  useReactTable,
+} from "@tanstack/react-table";
+import { css } from "@emotion/react";
+
+import { Flex, Icon, Icons, View } from "@arizeai/components";
+
+import { selectableTableCSS } from "@phoenix/components/table/styles";
+import { TimestampCell } from "@phoenix/components/table/TimestampCell";
+import { useStreamState } from "@phoenix/contexts/StreamStateContext";
+import { useTracingContext } from "@phoenix/contexts/TracingContext";
+
+import { IntCell, TextCell } from "../../components/table";
+import { TokenCount } from "../../components/trace/TokenCount";
+
+import { SessionsTable_sessions$key } from "./__generated__/SessionsTable_sessions.graphql";
+import { SessionsTableQuery } from "./__generated__/SessionsTableQuery.graphql";
+import { ProjectTableEmpty } from "./ProjectTableEmpty";
+import { SpanFilterConditionField } from "./SpanFilterConditionField";
+import { spansTableCSS } from "./styles";
+
+type SessionsTableProps = {
+  project: SessionsTable_sessions$key;
+};
+
+const PAGE_SIZE = 50;
+
+export function SessionsTable(props: SessionsTableProps) {
+  //we need a reference to the scrolling element for logic down below
+  const tableContainerRef = useRef<HTMLDivElement>(null);
+  const [sorting, setSorting] = useState<SortingState>([]);
+  const [filterCondition, setFilterCondition] = useState<string>("");
+  const navigate = useNavigate();
+  const { fetchKey } = useStreamState();
+  const { data, loadNext, hasNext, isLoadingNext, refetch } =
+    usePaginationFragment<SessionsTableQuery, SessionsTable_sessions$key>(
+      graphql`
+        fragment SessionsTable_sessions on Project
+        @refetchable(queryName: "SessionsTableQuery")
+        @argumentDefinitions(
+          after: { type: "String", defaultValue: null }
+          first: { type: "Int", defaultValue: 50 }
+        ) {
+          name
+          sessions(first: $first, after: $after, timeRange: $timeRange)
+            @connection(key: "SessionsTable_sessions") {
+            edges {
+              session: node {
+                id
+                sessionId
+                sessionUser
+                numTraces
+                startTime
+                endTime
+                firstInput {
+                  value
+                }
+                lastOutput {
+                  value
+                }
+                tokenUsage {
+                  prompt
+                  completion
+                  total
+                }
+              }
+            }
+          }
+        }
+      `,
+      props.project
+    );
+  const tableData = useMemo(() => {
+    return data.sessions.edges.map(({ session }) => session);
+  }, [data]);
+  type TableRow = (typeof tableData)[number];
+  const columns: ColumnDef<TableRow>[] = [
+    {
+      header: "session id",
+      accessorKey: "sessionId",
+      enableSorting: false,
+      cell: TextCell,
+    },
+    {
+      header: "session user",
```

Comment:
maybe just user here? not sure if there are any mocks to go off of but user seems a little more natural